types/chrome/index.d.ts

```diff
@@ -3489,31 +3314,29 @@ declare namespace chrome.events {
          * Unregisters currently registered rules.
          * @param ruleIdentifiers If an array is passed, only rules with identifiers contained in this array are unregistered.
          * @param callback Called when rules were unregistered.
-         * If you specify the callback parameter, it should be a function that looks like this:
-         * function() {...};
          */
         removeRules(ruleIdentifiers?: string[], callback?: () => void): void;
         /**
          * Unregisters currently registered rules.
          * @param callback Called when rules were unregistered.
-         * If you specify the callback parameter, it should be a function that looks like this:
-         * function() {...};
          */
         removeRules(callback?: () => void): void;
         /**
          * Registers rules to handle events.
          * @param rules Rules to be registered. These do not replace previously registered rules.
          * @param callback Called with registered rules.
-         * If you specify the callback parameter, it should be a function that looks like this:
-         * function(array of Rule rules) {...};
-         * Parameter rules: Rules that were registered, the optional parameters are filled with values.
          */
+        addRules(
+            rules: Rule[],
+            callback: (
```

Comment:
This appears to be a duplicate of the overlaod at line 3336. Prefer to just leave the callback parameter as optional rather than an overload that makes it optional. 

tl;dr> Can you make this callback optional, then remove the overload below?

Reply:
Good find, dropped